ghost/i18n/locales/da/ghost.json

```diff
@@ -1,27 +1,27 @@
 {
     "All the best!": "Alt det bedste!",
-    "Complete signup for {{siteTitle}}!": "",
+    "Complete signup for {{siteTitle}}!": "Gennemfør tilmelding på {{siteTitle}}!",
```

Comment:
What about: **Afslut din oprettelse på {{siteTitle}}!**

Reply:
I have rarely seen that sentence being used. _Oprettelse af din konto_, sure, but not that one. Have more encountered _tilmelding_